src/main/java/thePackmaster/cards/bladestormpack/AbstractBladeStormCard.java

```diff
@@ -0,0 +1,24 @@
+package thePackmaster.cards.bladestormpack;
+
+import thePackmaster.cards.AbstractPackmasterCard;
+
+/*REFS: AbstractGrandOpeningCard (grandopeningpack), AbstractEvenOddCard (evenoddpack),
+AbstractPickThemAllCard (pickthemallpack),*/
+public abstract class AbstractBladeStormCard extends AbstractPackmasterCard
+{
+    public AbstractBladeStormCard(String cardID, int cost, CardType type, CardRarity rarity, CardTarget target)
+    {
+        super(cardID, cost, type, rarity, target, "bladestorm");
+
+    }
+
+    @Override
+    public void upp() {
```

Comment:
Having this is redundant and I'd recommend deleting it. Trade Winds is the one card that doesn't do anything for `upp`, and it overrides `upp` itself (which is the better approach, since normally cards will need to override this).

Since you've already coded all the cards it doesn't make a significant difference, but deleting this is better in case you (or someone else) copies this as the basis for a new pack, or if you decide to deprecate one of the cards in this pack and make a new pack. Plus the general benefit of having Packmaster code consistently follow best practices and avoid redundant code.